### PR TITLE
gocritic comment should match error bugfix.

### PR DESCRIPTION
**Pull request by onuragtas**
Error;

gocritic check -enableAll ./...
./docs/docs.go:2:1: codegenComment: comment should match `Code generated .* DO NOT EDIT.` regexp
make: *** [critic] Error 1

This changes fix it.
Please approve pr.

### CODE REVIEW

**Review by hakanbaysal — COMMENTED** [review]
I tested this change. Thanks @onuragtas

**Review by hakanbaysal — APPROVED** [review]

**Review by hakanbaysal — DISMISSED** [review]
I tested this change. Thanks @onuragtas

File: gen/gen.go [hunk 1]
```diff
@@ -449,9 +449,9 @@ func (g *Gen) writeGoDoc(packageName string, output io.Writer, swagger *spec.Swa
 	return err
 }
 
-var packageTemplate = `// Package {{.PackageName}} GENERATED BY SWAG; DO NOT EDIT
-// This file was generated by swaggo/swag{{ if .GeneratedTime }} at
-// {{ .Timestamp }}{{ end }}
+var packageTemplate = `// Code generated by SWAG. DO NOT EDIT
+// Package {{.PackageName}}
+// {{ if .GeneratedTime }} at {{ .Timestamp }}{{ end }}
```
Comment (ubogdan) [inline gen/gen.go:454]: The phrase containing the generator package name and time stamp makes more sense to me.
```go
var packageTemplate = `// Code generated by swaggo/swag{{ if .GeneratedTime }} at {{ .Timestamp }}{{ end }}. DO NOT EDIT
// Package {{.PackageName}}`
```

**Review by ubogdan — CHANGES_REQUESTED** [review]

**Comment by ubogdan** [comment]
Also, please run the generator against the example folder .

File: gen/gen.go (hunk 1 above)
Comment (onuragtas) [reply gen/gen.go]: I changed it, can you check?

**Pull request merged**